libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Dialogs/Microsoft.AdaptiveSkillDialog.schema

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
This needs to be folded in the sdk.schema. See tooling under schemas folder.

Reply:
done